src/client/chunk_closure.cpp

```diff
@@ -238,6 +238,10 @@ void ClientClosure::Run() {
             OnEpochTooOld();
             break;
 
+        case CHUNK_OP_STATUS::CHUNK_OP_STATUS_READONLY:
+            OnReadOnly();
+            break;
```

Comment:
When the space is full, the client needs to retry

Reply:
ok.